.github/workflows/test.yml

```diff
@@ -18,46 +18,78 @@ jobs:
   # This workflow contains a single job called "build"
   build:
     # The type of runner that the job will run on
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        julia-version: ['1.6']
+        os: [ubuntu-latest]
+        algorithm: ["AFPRegressor", "AIFeynman", "BSRRegressor", "DSRRegressor", "EHCRegressor", "FFXRegressor", "FEATRegressor", "FE_AFPRegressor", "EPLEXRegressor", "GPGOMEARegressor", "gplearn", "ITEARegressor", "MRGPRegressor", "OperonRegressor", "PySRRegressor", "sembackpropgp"]
```

Comment:
would be more intelligible to store as a list like 
```
algorithms:
  - name: AFPRegressor
  - name: AIFeynman
  - ...
```

if possible

Reply:
Good idea